src/convert.rs

```diff
@@ -967,6 +990,18 @@ impl From<TransactionError> for tx_by_addr::TransactionError {
                         },
                     })
                 }
+                TransactionError::InsufficientFundsForRent { account_index } => {
```

Comment:
In v1.10.31 of validator:

            },
            transaction_details: match transaction_error {
                TransactionError::DuplicateInstruction(index) => {
                    Some(tx_by_addr::TransactionDetails {
                        index: index as u32,
                    })
                }
                TransactionError::InsufficientFundsForRent { account_index } => {
                    Some(tx_by_addr::TransactionDetails {
                        index: account_index as u32,
                    })
                }
                _ => None,
            },